app/Controllers/Admin/Settings/TrustLevelsController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Controllers\Admin\Settings;
+
+use App\Controllers\AdminController;
+use InvalidArgumentException;
+
+class TrustLevelsController extends AdminController
+{
+    /**
+     * Display the trust levels settings page.
+     */
+    public function index()
+    {
+        if ($this->request->getMethod() === 'post') {
```

Comment:
```suggestion
        if ($this->request->is('post')) {
```
Available since version 4.3. Of course, I know that you are not interested in `is()`.

Reply:
Never said I wasn't interested. I knew getMethod had a deprecation but couldn't remember what the alternate was. And completely forgot about `is()` so thanks for reminding me!